src/components/attendee/AttendeeLayout.tsx

```diff
@@ -21,20 +22,30 @@ export const AttendeeLayout: React.FC = () => {
     };
   }, []);
 
+  if (!userId) {
+    return <p>Loading</p>;
+  }
+
   return (
-    <VideoHuddleProvider>
-      <header></header>
-      <main>
-        <section className={styles.Space}>
-          <VenuePage />
-        </section>
-        <div className={styles.LayerUi}>
-          <ChatContainer />
-          {userId && <VideoHuddle userId={userId} />}
-        </div>
-      </main>
-
-      <AttendeeFooter />
-    </VideoHuddleProvider>
+    <VideoCommsProvider userId={userId}>
+      <HuddleProvider>
+        <header></header>
+        <main>
+          <section className={styles.Space}>
+            <VenuePage />
+          </section>
+          <div className={styles.LayerUi}>
+            <ChatContainer />
+            {/*
+          TODO VideoHuddle being available depends on the context of the space we're in.
+          This probably needs more thought as to what API it provides.
+        */}
+            {userId && <VideoHuddle userId={userId} />}
```

Comment:
Any place in the context for these edge cases https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1644330820946409?thread_ts=1643886209.361969&cid=C01ER4M61TL ?